microshift_release_notes/microshift-4-17-release-notes.adoc

```diff
@@ -224,4 +224,18 @@ Issued: 28 January 2025
 
 Bug fixes and enhancements are listed in the link:https://access.redhat.com/errata/RHBA-2025:0682[RHBA-2025:0682] advisory. Release notes for bug fixes and enhancements are provided in this documentation. The images that are included in the update are provided by the {OCP} link:https://access.redhat.com/errata/RHSA-2025:0654[RHSA-2025:0654] advisory.
 
-See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
\ No newline at end of file
+See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
+
+[id="microshift-4-17-27-dp_{context}"]
+=== RHBA-2025:0682 - {microshift-short} 4.17.27 bug fix and enhancement advisory
+
+Issued: 30 April 2025
+
+{product-title} release 4.17.27 is now available, see the link:https://access.redhat.com/errata/RHBA-2025:4217[RHBA-2025:4217] advisory. Release notes for bug fixes and enhancements are provided in this documentation. The images that are included in the update are provided by the {OCP} link:https://access.redhat.com/errata/RHSA-2025:4204[RHSA-2025:4204] advisory.
+
+See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
+
+[id="ocp-4-17-27-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, an IPv6 configuration for {microshift-short} was created in the system at first start when an IP address was not configured. This was caused by an erroneous retrieval of the default node IP address because of order in which the interfaces were listed. This misconfiguration of the default IP address could result in the service being unreachable. With this release, the correct node IP address is retrieved first. (link:https://issues.redhat.com/browse/OCPBUGS-52861[OCPBUGS-52861])
```

Comment:
```suggestion
* Previously, an IPv6 configuration for {microshift-short} was created in the system at first start when an IP address was not configured. This was caused by an erroneous retrieval of the default node IP address because of the order in which the interfaces were listed. Consequentially, the misconfiguration resulted in services being unreachable. With this release, the correct node IP address is retrieved first. (link:https://issues.redhat.com/browse/OCPBUGS-52861[OCPBUGS-52861])
```